### PR TITLE
ignition_collection.dsl: add bottle install jobs

### PR DESCRIPTION
**Pull request by scpeters**
This is a follow up to #209 that adds relevant `install_bottle` jobs to the blueprint and citadel jenkins views.

* [Build Status](https://build.osrfoundation.org/job/_dsl_ignition_collection/196/) https://build.osrfoundation.org/job/_dsl_ignition_collection/196/
* https://build.osrfoundation.org/view/ign-blueprint/
* https://build.osrfoundation.org/view/ign-citadel/

### CODE REVIEW

**Comment by scpeters** [comment]
I suppose we could add `install_bottle` jobs for blueprint and citadel to this file as well

**Comment by scpeters** [comment]
we could add ign-cmake2 install jobs to the collections as well

**Comment by scpeters** [comment]
> I suppose we could add install_bottle jobs for blueprint and citadel to this file as well

done in 72da583

[Build Status](https://build.osrfoundation.org/job/_dsl_ignition_collection/197/) https://build.osrfoundation.org/job/_dsl_ignition_collection/197/

> we could add ign-cmake2 install jobs to the collections as well

done in 60f2db3

File: jenkins-scripts/dsl/ignition_collection.dsl [hunk 1]
```diff
@@ -41,55 +41,71 @@ ignition_collection_jobs =
...
+        'ignition_rendering2-install_bottle-homebrew-amd64',
         'ignition_sensors-ci-ign-sensors2-bionic-amd64',
         'ignition_sensors-ci-ign-sensors2-homebrew-amd64',
         'ignition_sensors2-install-pkg-bionic-amd64',
+        'ignition_sensors2-install_bottle-homebrew-amd64',
         'ignition_tools-ci-ign-tools0-bionic-amd64',
         'ignition_tools-ci-ign-tools0-homebrew-amd64',
         'ignition_tools-ci-ign-tools0-windows7-amd64',
         'ignition_tools-install-pkg-bionic-amd64',
+        'ignition_tools1-install_bottle-homebrew-amd64',
```
Comment (chapulina) [inline jenkins-scripts/dsl/ignition_collection.dsl:103]: Blueprint uses `tools0`

Comment (j-rivero) [reply jenkins-scripts/dsl/ignition_collection.dsl]: I've change it in 76925e2

**Comment by j-rivero** [comment]
The PR looks good to me. We have some unstable builds generated by this PR (like the citadel bottle) but it's out of scope of this PR to solve that. Thanks Steve, Louise.

**Pull request merged**